cmd/kubernetesCmd.go

```diff
@@ -150,6 +150,37 @@ func init() {
 		"",
 		"The AWS region that the custom s3 bucket is in",
 	)
+	kubernetesCmd.PersistentFlags().StringVar(
+		&config.CustomAzureUploadBlobContainerName,
+		"custom_azure_blob_container_name",
+		"",
+		"The Azure blob container name the metrics-agent will upload data to. Default is an empty string "+
+			"which will not upload to custom azure location",
```

Comment:
```suggestion
		"The Azure blob container name the metrics-agent will upload data to.",
```

Providing information about a default that doesn't work is confusing imo